test/features/resource_validation.feature

```diff
@@ -103,34 +86,29 @@ Feature: Resource validation
         Then I should see "Test valid CSV update"
         Then I click the link with text that contains "Test valid CSV update"
         And I click the link with text that contains "Manage"
-        And I attach the file "valid.csv" to "upload"
+        And I attach the file "test.csv" to "upload"
+        And I attach the file "test_schema.json" to "schema_upload"
         And I press the element with xpath "//button[contains(@class, 'btn-primary')]"
-
-        When I wait for 5 seconds
         And I click the link with text that contains "Test valid CSV update"
         Then I should see "Validation status"
         And I should see "success"
         And I should not see "failure"
         And I should see a validation timestamp
 
 
-    Scenario: As an editor, I can update a resource with an invalid CSV and see a failure status
+    Scenario: As an editor, I updating a resource with an invalid CSV and see a validation error
```

Comment:
"I updating" is incorrect grammar. It was correct before, "I can update".

Reply:
+

Reply:
This will be fixed and pushed by Alex shortly.